Automattic/pocket-casts-ios
# Discover Collection View: Add collection view and view controller cells

| 📘 Part of: #2104 |
|:---:|

This adds the collection view for Discover.

**Categories and regions are not hooked up yet**

https://github.com/user-attachments/assets/022645d0-5fbb-4f60-911c-823d11543cd2

## To test

* Enable the `discoverCollectionView` feature flag
* Restart the app
* Visit the Discover tab
* Verify that everything except for categories and region switching works. Tap on a few podcasts, try subscribing, searching, and scroll around for example.

## Checklist
